### PR TITLE
Auto focus Message input field on workspace invite member page

### PR DESCRIPTION
**Pull request by PrashantMangukiya**
@hayata-suenaga @mollfpr PR is ready for review.

### Details
Whenever we select member from **Invite member**  screen, and tap **Next** button it will show **Add message** screen. On this screen there is a **Message** input field that was not auto focused until now. So in this pr we added logic to autofocus the Message input field.

### Fixed Issues
$ https://github.com/Expensify/App/issues/18371
PROPOSAL: https://github.com/Expensify/App/issues/18371#issuecomment-1533912665

### Tests
1. Go to Settings > Workspace > Manage Members
2. Tap **Invite** button (Green)
3. Select member(s) to invite
4. Tap **Next** button
5. **Verify that Message field is auto-focused**

- [x] Verify that no errors appear in the JS console

### Offline tests
1. Go to Settings > Workspace > Manage Members
2. Tap **Invite** button (Green)
3. Select member(s) to invite
4. Tap **Next** button
5. **Verify that Message field is auto-focused**

### QA Steps
1. Go to Settings > Workspace > Manage Members
2. Tap **Invite** button (Green)
3. Select member(s) to invite
4. Tap **Next** button
5. **Verify that Message field is auto-focused**

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

#### Chrome

https://github.com/Expensify/App/assets/7823358/e8d6d602-1443-4649-a319-1f495b55c7a6

#### Safari

https://github.com/Expensify/App/assets/7823358/aeec567e-41b1-4d75-acc2-543517e634e7

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://github.com/Expensify/App/assets/7823358/cbc158ef-104f-4f8c-a9a0-241f8a68563b

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://github.com/Expensify/App/assets/7823358/56e0122a-66ce-49c7-be42-340f805f3e3d

</details>

<details>
<summary>Desktop</summary>

https://github.com/Expensify/App/assets/7823358/ceec3e58-98b9-4e6c-b21b-b15ace895acc

</details>

<details>
<summary>iOS</summary>

https://github.com/Expensify/App/assets/7823358/f7f767ba-30a9-4447-a940-e8a0e22c1e72

</details>

<details>
<summary>Android</summary>

https://github.com/Expensify/App/assets/7823358/4aa30ff3-9085-4f42-87ab-71d18a3dbda2

</details>

### CODE REVIEW

**Comment by mollfpr** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



https://github.com/Expensify/App/assets/25520267/a6119fc9-0613-440f-81db-65e182d87e80


</details>

<details>
<summary>Mobile Web - Chrome</summary>



https://github.com/Expensify/App/assets/25520267/c9fd137e-38e9-4080-87eb-ec07847c1dce


</details>

<details>
<summary>Mobile Web - Safari</summary>



https://github.com/Expensify/App/assets/25520267/6b606e1e-7e11-4834-b938-1d673c7c877f


</details>

<details>
<summary>Desktop</summary>



https://github.com/Expensify/App/assets/25520267/c32835aa-c5f8-4ae2-85eb-9798d8fc36e6


</details>

<details>
<summary>iOS</summary>



https://github.com/Expensify/App/assets/25520267/7a46cdc4-b8bd-4143-839c-0db5f92c1dde


</details>

<details>
<summary>Android</summary>



https://github.com/Expensify/App/assets/25520267/fe12c6b5-1d82-43ee-9240-4afb962d9e03


</details>

**Comment by mollfpr** [comment]
@PrashantMangukiya @hayata-suenaga Coming from my curiosity, should we also make the input focus after refresh?

**Comment by PrashantMangukiya** [comment]
> @PrashantMangukiya @hayata-suenaga Coming from my curiosity, should we also make the input focus after refresh?

I think it is also possible option. If we have to do this then we have to remove code from `onEntryTransitionEnd={() => {}}` and put within componentDidMount as shown code below. It will work for both refresh and regular navigation. I tested it and it is working fine in all platform.
```javascript
componentDidMount() {
    this.focusTimeout = setTimeout(() => {
        this.welcomeMessageInputRef.focus();
        // Below condition is needed for web, desktop and mweb only, for native cursor is set at end by default.
        if (this.welcomeMessageInputRef.value && this.welcomeMessageInputRef.setSelectionRange) {
            const length = this.welcomeMessageInputRef.value.length;
            this.welcomeMessageInputRef.setSelectionRange(length, length);
        }
    }, CONST.ANIMATED_TRANSITION);
}

componentWillUnmount() {
    if (!this.focusTimeout) {
        return;
    }
    clearTimeout(this.focusTimeout);
}
```

**Comment by mollfpr** [comment]
@PrashantMangukiya Let's try your suggestion above.

**Comment by PrashantMangukiya** [comment]
@mollfpr Pushed updated code to support focus after refresh. Please let me know if anything else is needed. It will be great if we can merge PR asap, so eligible for timeline bonus. Thank you.

**Review by mollfpr — APPROVED** [review]
Nice work @PrashantMangukiya! It tests well 👍

All yours @hayata-suenaga

**Review by mollfpr — APPROVED** [review]
Trigger again the PR checklist check.

**Comment by PrashantMangukiya** [comment]
@hayata-suenaga It will be great if we can merge PR, so it will be eligible for timeline bonus. Thank you.

**Comment by hayata-suenaga** [comment]
@PrashantMangukiya I left a comment above. It's strongly discouraged to use `setTimeOut` in the codebase. do you think there is any way we can avoid it?

**Comment by PrashantMangukiya** [comment]
@hayata-suenaga As we have to set focus during browser refresh also, here time out is needed to set focus during componentDidMount. 

Also our BaseTextInput component using timeout to set focus on `componentDidMount`.
https://github.com/Expensify/App/blob/896ad77e328e044df17f0df3b563f2b017369d82/src/components/TextInput/BaseTextInput.js#L63-L66

So here If we have to avoid timeout then we can go with original proposed solution, but that will not focus during page refresh so we implemented this. Thank you.

**Comment by hayata-suenaga** [comment]
@PrashantMangukiya with `setTimeout`, are we waiting for some kind of animations to finish?

I'd appreciate it if you could expand on that a little bit

**Comment by PrashantMangukiya** [comment]
@hayata-suenaga by setting timeout we can make sure that it will finish transition and then set focus. Previously we use the solution based on the onEnteryTransitionEnd() as per my initial [proposal here](https://github.com/Expensify/App/issues/18371#issuecomment-1533912665)  But within that, focus was not set when browser refresh. So to make focus work during browser refresh we used timeout based solution. So this solution is working in both situation i.e.normal navigation and browser refresh. So this is my thought to use this solution. 

Please let me know if you have different idea that will achieve the same goal. Thank you.

**Comment by hayata-suenaga** [comment]
Thank you for the detailed explanation. Although I understand the reasoning, I'm curious if anyone has a solution that doesn't use `setTimeout` (`setTimeout` has caused troubles in the past and I'm really hesitant to push codes that contain `setTimeout`

so I opened up a [thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1684263976356599) in #expensify-open-source Slack channel.

**Comment by hayata-suenaga** [comment]
from the above Slack thread, we determined that we have no other good choice but to use `setTimeout` for now.

**Review by hayata-suenaga — APPROVED** [review]
LGTM!

**Comment by hayata-suenaga** [comment]
@PrashantMangukiya, the QA for this PR has failed

the original issue has not been on mWeb Safari and iOS. can you investigate?

**Comment by PrashantMangukiya** [comment]
@mvtglobally @hayata-suenaga I tested it, this issue is fixed for iOS and Mweb as shown video here. 

### MWeb Safari 
https://github.com/Expensify/App/assets/7823358/be43b58a-0701-43f0-86f4-018c21cc9f0f 

### iOS
https://github.com/Expensify/App/assets/7823358/f086229e-6688-4df5-95f6-5bf1a2c573ee

**Comment by hayata-suenaga** [comment]
@PrashantMangukiya thank you for testing again. I'll update the QA team if they can redo the tests again

**Comment by mvtglobally** [comment]
@hayata-suenaga team is still able to repro https://github.com/Expensify/App/assets/43995119/d442df5f-81d8-4871-86a8-ce27a3af0b0d

**Comment by hayata-suenaga** [comment]
@PrashantMangukiya can you test it again with staging changes?

**Comment by mvtglobally** [comment]
Issue is not repro on IOS 16.4 
Only MWeb Safari is the issue

**Comment by PrashantMangukiya** [comment]
@mvtglobally For Staging Mweb Safari it is not reproduce in emulator.  I can see you have real device, where it shows green bottom line (due to focus in input) but does not show cursor. This is strange behaviour. Not any idea at present, I will investigate more about this.  cc: @hayata-suenaga 

https://github.com/Expensify/App/assets/7823358/24613072-6031-40d3-880a-ac1cf11f7655

**Comment by PrashantMangukiya** [comment]
@hayata-suenaga @mvtglobally I did some investigation on MWeb focus issue for real device and found out that this issue is not specific to our PR, but it is in many places in our App (maybe due to faulty TextInput component) having same behaviour i.e. TextInput focused but cursor does not show. Below are few videos recorded on Real Device MWeb Safari.

<details>
<summary>New Contact Method</summary>

https://github.com/Expensify/App/assets/7823358/2671a7e1-26e9-4658-aac9-fcba8167afdc

</details>

<details>
<summary>Change Password</summary>

https://github.com/Expensify/App/assets/7823358/36b22f09-7f25-4141-bab6-b4a5f772b36c

</details>

<details>
<summary>Add PayPal</summary>

https://github.com/Expensify/App/assets/7823358/6e52ca45-543a-4093-9f86-ef5edac4b5df

</details>

<details>
<summary>New Chat</summary>

https://github.com/Expensify/App/assets/7823358/08a005d4-7971-49c2-be0f-964cf2088763

</details>

<details>
<summary>New Group</summary>

https://github.com/Expensify/App/assets/7823358/45970c5e-26fb-43c1-a4a7-c8e928583572

</details>

<details>
<summary>Send Money Description</summary>

https://github.com/Expensify/App/assets/7823358/45968dc8-8f0c-49b6-9e96-d38794696398

</details>

<details>
<summary>Request Money Description</summary>

https://github.com/Expensify/App/assets/7823358/3079ab1d-429d-447c-83e8-2729a0e173fa

</details>

<details>
<summary>Split Bill Description</summary>

https://github.com/Expensify/App/assets/7823358/ef4e05c0-1354-4762-90bd-367d0118efef

</details>

<details>
<summary>Assign Task Title</summary>

https://github.com/Expensify/App/assets/7823358/57a953cb-4e01-489a-94a8-81923ab9804a

</details>

<details>
<summary>Assign Task Description</summary>

https://github.com/Expensify/App/assets/7823358/6992802c-55df-4cdc-b1d0-942e55d38880

</details>

I think we can create a new issue for this.

**Comment by hayata-suenaga** [comment]
@PrashantMangukiya can you report the bug to #expensify-open-source channel?

this is very weird... the text input is focused but the cursor is not being shown and the keyboard is not shown up...

**Comment by PrashantMangukiya** [comment]
@hayata-suenaga Should I report this to **#expensify-open-source** OR **#expensify-bugs** channel? Generally we report bug on #expensify-bugs channel so I am asking to verify it. Thank you.

File: src/pages/workspace/WorkspaceInviteMessagePage.js [hunk 1]
```diff
@@ -78,6 +78,17 @@ class WorkspaceInviteMessagePage extends React.Component {
...
     }
 
+    componentDidMount() {
+        this.focusTimeout = setTimeout(() => {
+            this.welcomeMessageInputRef.focus();
+            // Below condition is needed for web, desktop and mweb only, for native cursor is set at end by default.
+            if (this.welcomeMessageInputRef.value && this.welcomeMessageInputRef.setSelectionRange) {
+                const length = this.welcomeMessageInputRef.value.length;
+                this.welcomeMessageInputRef.setSelectionRange(length, length);
+            }
```
Comment (Santhosh-Sellavel) [inline src/pages/workspace/WorkspaceInviteMessagePage.js:88]: This is caused a regression here https://github.com/Expensify/App/issues/20875#issuecomment-1594421713.

Why do we need this?

Comment (hayata-suenaga) [reply src/pages/workspace/WorkspaceInviteMessagePage.js:88]: cc: @mollfpr @PrashantMangukiya

Comment (mollfpr) [reply src/pages/workspace/WorkspaceInviteMessagePage.js:88]: Why is https://github.com/Expensify/App/issues/20875 a regression of this?

On the native the cursor is set at the end of the text, so we want the same behavior for the web and desktop.

**Pull request merged**